languages/java/exercises/concept/switch-statement/.docs/after.md

```diff
@@ -0,0 +1,134 @@
+Starting with Java 14 (available as a preview before in Java 12 and 13) it is possible to use the "enhanced" switch implementation.
+
+1. You have the possiblity to assign multiple value in a sigle case.
+   Instead of :
+
+   ```java
+   switch (number) {
+       case 1:
+           //do stuff
+           break;
+       case 2:
+           //do different stuff
+           break;
+       case 3:
+           //do same stuff as case 1
+           break;
+       (...)
+   }
```

Comment:
Theoretically you could already use fall-through, right?

```suggestion
   switch (number) {
       case 1:
       case 2:
           //do different stuff
           break;
       case 3:
           //do same stuff as case 1
           break;
       (...)
   }
```

I would suggest also explaining the fall-through functionality of a switch statement.

Reply:
I adapted the exemple to use fall-thought